UrlManager.php

```diff
@@ -134,6 +134,18 @@ class UrlManager extends BaseUrlManager
      */
     public $languageParam = 'language';
 
+    /**
+     * @var string $_SERVER key set by apache mod_geoip
+     */
+    public $geoipServerFieldName = 'HTTP_X_GEO_COUNTRY';
+
+    /**
+     * @var array list of countries for given language code
+     * e.g. 'ru' => ['RUS','AZE','ARM','BLR','KAZ','KGZ','MDA','TJK','TKM','UZB','UKR']
+     * will set app language to ru for countries listed above
+     */
+    public $geoipLanguageCountries = [];
```

Comment:
I think the correct camel case name should be `geoIpLanguageCountries` (with upper case I)